Dockerfile

```diff
@@ -7,7 +7,9 @@ FROM bash:${bashver}
 RUN apk add --no-cache parallel && \
     mkdir -p ~/.parallel && touch ~/.parallel/will-cite
 
-RUN ln -s /opt/bats/bin/bats /usr/sbin/bats
-COPY . /opt/bats/
+WORKDIR /opt/bats
+RUN ln -s /opt/bats/bin/bats /usr/local/bin/bats
```

Comment:
Another potentially breaking change I think?

Reply:
No, because it is in PATH it isn't a breaking change.